samples/DotNet/Microsoft.Azure.ServiceBus/SendReceiveUsingMessageSenderReceiver/readme.md

```diff
@@ -0,0 +1,141 @@
+# Get started sending and receiving messages from Service Bus queues using MessageSender and MessageReceiver
```

Comment:
The difference is mentioned in detail under the ## What will be accomplished section below?